### ISSUE TITLE
Overlapping branch paths if only one commit is present

### ISSUE THREAD

**Issue by lhassis** [nicoespeon/gitgraph.js#396]
If we fork a new branch (name it develop) and add to it a first commit only after a new one is created on the previous branch (name it master), the "develop" branch has its path drawn overlaping the "master" branch path. But if we add a new commit to develop or a merge into master, the paths are shown correctly separated.

Overlapping scenario
![image](https://user-images.githubusercontent.com/49009472/100445554-0e653300-308c-11eb-84ff-4ae6b17fb7b9.png)

Correct drawing with merge
![image](https://user-images.githubusercontent.com/49009472/100445614-289f1100-308c-11eb-9f6e-94301cf180b2.png)

Correct drawing with a second commit
![image](https://user-images.githubusercontent.com/49009472/100445662-3d7ba480-308c-11eb-8591-069ec74fee83.png)

Is there a way to make it draw separated paths on the first scenario?

**Comment by lhassis** [comment]
As an update for this issue, we found out that it only happens when we use de vertical reversed orientation. Bellow an example without that
![image](https://user-images.githubusercontent.com/49009472/100614612-25f32480-32f5-11eb-8e62-27e3f213b84d.png)

**Comment by lhassis** [comment]
I saw that the same issue was opened on https://github.com/nicoespeon/gitgraph.js/issues/282, so I'closing this one.
